pyquil/tests/test_device.py

```diff
@@ -1,52 +1,126 @@
 from collections import OrderedDict
 
 import numpy as np
+import pytest
 
-from pyquil.device import KrausModel, ISA, Qubit, Edge, NoiseModel
-
-ARCH_QPU = {
-    "id": {
-        "name": "Mixed architecture chip",
-        "version": "0.0",
-        "timestamp": "20180104103600"
-    },
-    "logical-hardware": [
-        [
-            {
-                "qubit-id": 0,
-                "type": "Xhalves"
-            },
-            {
-                "qubit-id": 1
-            },
-            {
-                "qubit-id": 2
-            }
-        ],
-        [
-            {
-                "action-qubits": [0, 1],
-                "type": "CZ"
-            },
-            {
-                "action-qubits": [1, 2],
-                "type": "ISWAP"
-            },
-            {
-                "action-qubits": [2, 0],
-                "type": "CPHASE"
-            }
+from pyquil.device import Device, KrausModel, ISA, Qubit, Edge, NoiseModel
+
+DEVICE_FIXTURE_NAME = 'mixed_architecture_chip'
+
+
+@pytest.fixture
+def isa_dict():
+    return {
+        "id": {
+            "name": DEVICE_FIXTURE_NAME,
+            "version": "0.0",
+            "timestamp": "20180104103600"
+        },
+        "logical-hardware": [
+            [
+                {
+                    "qubit-id": 0,
+                    "type": "Xhalves"
+                },
+                {
+                    "qubit-id": 1
+                },
+                {
+                    "qubit-id": 2
+                }
+            ],
+            [
+                {
+                    "action-qubits": [0, 1],
+                    "type": "CZ"
+                },
+                {
+                    "action-qubits": [1, 2],
+                    "type": "ISWAP"
+                },
+                {
+                    "action-qubits": [2, 0],
+                    "type": "CPHASE"
+                }
+            ]
         ]
-    ]
-}
+    }
+
+
+@pytest.fixture
+def isa(isa_dict):
```

Comment:
because this fixture is only used in `test_device` i think it would be clearer to just pass the dictionary fixture to that test and build the `ISA` in the test with `ISA.from_dict`